test/functional/toolchains/qbs/test_qbs.py

```diff
@@ -11,4 +21,86 @@ def test_api_qbs_create_lib(shared):
     client = TestClient()
     client.run("new qbs_lib -d name=hello -d version=1.0")
     client.run("create . -o:h &:shared={shared}".format(shared=shared))
-    assert "compiling hello.cpp" in client.out
\ No newline at end of file
+    assert "compiling hello.cpp" in client.out
+
+
+@pytest.mark.tool("qbs")
+def test_qbs_all_products():
+    client = TestClient()
+
+    context = {
+        "name": "hello",
+        "version": "2.0",
+        "package_name": "hello"
+    }
+
+    conanfile = textwrap.dedent('''
+    import os
+
+    from conan import ConanFile, tools
+    from conan.tools.qbs import Qbs
+    from conan.tools.files import copy, collect_libs
+
+    class Recipe(ConanFile):
+        name = "hello"
+        version = "1.0"
+
+        exports_sources = "*.cpp", "*.h", "*.qbs"
+        settings = "os", "compiler", "arch"
```

Comment:
In general, most packages have ``build_type`` as setting. I don't know if ``Qbs`` has some model of build-types, but it would be good to have the default templates, tests, etc. to use build-types too.

Reply:
We do, yes.